packages/core/data-transfer/src/strapi/remote/handlers/pull.ts

```diff
@@ -167,6 +167,16 @@ export const createPullController = handlerControllerFactory<Partial<PullHandler
     const stream = this.streams?.[stage];
 
     const batchLength = () => Buffer.byteLength(JSON.stringify(batch));
+
+    const maybeConfirm = async (data: any) => {
+      try {
+        await this.confirm(data);
+      } catch (error) {
+        // Handle the error, log it, or take other appropriate actions
+        console.error('Error during confirmation:', error);
```

Comment:
Could we make this a bit more specific? Like the fact it comes from DTS and killed it or smth?
Otherwise it might be hard for users to understand the source of this log

Reply:
Oops, yeah, and it should also try to send it through strapi.log rather than console. Good catch!

Reply:
Ok, added that!

Next step for the backlog: we need to make the diagnostics reporter also log warnings and errors to strapi logger whenever it's available; right now it doesn't, which is why I'm now logging to it directly like we do elsewhere